### PR TITLE
add functionality to delete note

### PR DESCRIPTION
**Pull request by abhinandam**
Adding Functionality to delete a note from the default notebook or a specific notebook if the NotebookName is passed in. 

We should really start adding tests for this as well!

### CODE REVIEW

File: models/db.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ type Datastore interface {
 	GetNotebook(notebookTitle string) (Notebook, error)
 	GetAllNotebooks() ([]Notebook, error)
 	AddNote(notebookTitle string, note Note) error
+	DeleteNote(notebookName string, noteId uint64) error
```
Comment (abhinandam) [inline models/db.go:13]: I kept the parameter convention the same, but why even pass in a `uint64` if the argument from the command line is a string and we end up converting it to a `[]byte` anyway?

**Review by ifennna — APPROVED** [review]
This is great! Thanks!

File: models/db.go (hunk 1 above)
Comment (ifennna) [reply models/db.go:13]: Good call. The id can just be a `[]byte` instead.

**Comment by ifennna** [comment]
Also agree with the tests. I haven't had too much time to work on this recently, but that should change in the near future.

**Pull request merged**